pkcs7/tests/content_tests.rs

```diff
@@ -139,6 +143,9 @@ fn decode_signed_scep_example() {
         }
         _ => panic!("expected ContentInfo::SignedData(Some(_))"),
     }
+
+    let mut buf = vec![0u8; bytes.len()];
+    let encoded_content = encode_content_info(&content, &mut buf);
```

Comment:
It's complaining about the unused variable:

```suggestion
    encode_content_info(&content, &mut buf);
```